mqtt/mqtt-broker/src/broker.rs

```diff
@@ -428,7 +430,7 @@ where
             }
         }
 
-        debug!("connect handled.");
+        info!("connect handled.");
```

Comment:
Again - with multiple clients, I feel it is useful to include the client_id as well.